martin-mbtiles/src/bin/main.rs

```diff
@@ -56,6 +69,27 @@ async fn main() -> Result<()> {
         Commands::MetaGetValue { file, key } => {
             meta_get_value(file.as_path(), &key).await?;
         }
+        Commands::Copy {
+            src_file,
+            dst_file,
+            verbose,
+            min_zoom,
+            max_zoom,
+            zoom_levels,
+        } => {
+            let mut tile_copier_options =
```

Comment:
please rename this var for readability - a bit too long for such a short usage.... maybe `opt` or `copy_opts` or ...?